package/MDAnalysis/lib/pkdtree.py

```diff
@@ -202,7 +204,7 @@ def search(self, centers, radius):
         self._indices = np.asarray(unique_int_1d(self._indices))
         return self._indices
 
-    def get_indices(self):
+    def get_indices(self) -> npt.NDArray:
```

Comment:
according to the docstring self._indices should be a list but doesn't np.asarray return NDArray